homeassistant/components/reolink/sensor.py

```diff
@@ -75,6 +75,21 @@ class ReolinkHostSensorEntityDescription(
     ),
 )
 
+HDD_SENSORS = (
+    ReolinkSensorEntityDescription(
+        key="storage",
+        cmd_key="GetHddInfo",
+        native_unit_of_measurement=PERCENTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        value=lambda api, idx: api.hdd_storage(idx),
+        supported=lambda api, idx: (
+            api.supported(None, "hdd") and api.hdd_type(idx) == "HDD"
```

Comment:
```suggestion
            api.supported(None, "hdd") and api.hdd_type(idx) in ("HDD", "SD")
```

It's probably left over on the simplify commit.

Reply:
Thanks for pointing this out, it was indeed a oversight.
I have corrected it now.